tools/runtime-stats-monitor.md

```diff
@@ -0,0 +1,215 @@
+:::
+
+## Adding the Runtime Network Stats Monitor to a Scene
+
+To add the RNSM to your scene, you must add it as a component to an object:
+
+1. Select an object in your scene. <- SHOULD AN OBJECT BE MADE SPECIFICALLY FOR THE RNSM? LIKE BEST PRACTICE-WISE? EMPTY OBJECT OR A SPECIFIC TYPE
+2. Add the RNSM commponent by either:
+   1. Selecting **Add Component** in the **Inspector** tab, navigate to **Netcode** and select **RuntimeNetStatsMonitor**.
+   OR
+   2. From the top menu bar, go to **Component** > **Netcode** > **RuntimeNetStatsMonitor**.
+
+## Configuring
+
+You can configure the RNSM to your liking with its [Display Elements](#display-elements), [Panel Settings](#panel-settings), and [Styling](#styling-the-runtime-network-stats-monitor). Each of these settings can be found from the **Inspector** tab of the RNSM component. The RNSM package comes with default settings for each option that can be used "as is" or as guidance for creating your own.
+
+:::note
+The default setting for the Display Elements, Panel Settings, and Styling are greyed out and not adjustable by default. To make modifications or build your own, you will need to create your own configuration presets. Instructions for each customization are below.
+:::
+
+### Display Elements
+
+Each Display Element displays one or more stats that can be added, removed, or modified under the element's **Stats** field.
+
+To use the default Display Elements:
+
+1. It may already be the default in the **Configuration** field of the **Inspector** tab of the RNSM component.
+
+OR
+
+1. From the **Project** tab, go to **Packages** > **Multiplayer Tools** > **NetStatsMonitor** > **Assets** > **Configurations**. The default configuration for the RNSM is in this folder.
+2. Select the `GameObject` with the RNSM component so that it appears in the **Inspector** tab.
+3. Drag the default configuration asset from the **Project** tab to the **Configuration** field under the **RuntimeNetStatsMonitor** component of your `GameObject`.
+
+To configure your own Display Elements presets:
+
+1. Select the **Assets** folder under the **Project** tab.
+2. Right-click in the **Assets** folder, then navigate to **Create** > **Multiplayer** and click on **NetStatsMonitorConfiguration**.
+3. The configuration script appears under the **Assets** folder and also opens in the **Inspector** tab.
+4. From the **Inspector** tab, you can use the + or - symbols to add each Display Element you want to include in this preset. Each field is configurable from here.
+5. Once configured, you can add your Display Elements configuration preset to your RNSM `GameObject` by selecting the the RNSM `GameObject` to appear in the **Inspector** tab.
+6. Your **Assets** folder should still be visible in the **Project** tab, so select your newly created configuration preset to drag and drop over the **Configuration** field under the **RuntimeNetStatsMonitor** component in the **Inspector** tab.
+7. **Save** your scene.
```

Comment:
This might not be ideal for the user, as it starts them with an empty configuration. I worry that an empty configuration would be more difficult for them to get started customizing, than if they have an example to follow.

We could suggest to them to copy the default as a jumping off point, from the location you mentioned above: **Packages** > **Multiplayer Tools** > **NetStatsMonitor** > **Assets** > **Configurations**.

Either approach works, but I worry it may be harder for users to get started customizing this without an example.